### ISSUE TITLE
Add example generated docs

### ISSUE THREAD

**Issue by JoshBrudnak** [JoshBrudnak/Lojidoc#5]
Use `lojidoc` to generate documentation for use as an example of what lojidoc output looks like.
Then add a brief description and link to the generated documentation to the readme.

The generated documentation should be put in the `examples` directory.

**Comment by jdemieville** [comment]
I can do this!

**Comment by jdemieville** [comment]
I am getting an error when I run lojidoc. It says that it is generating documentation from the appropriate file, then "Keyword not supported: this" pops up 8 times followed by "thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 123, kind: Other, message: "The filename, directory name, or volume label syntax is incorrect." }', libcore\result.rs:945:5.

**Comment by jdemieville** [comment]
There is a note: Run with `RUST_BACKTRACE=1` for a backtrace.
It does make the appropriate folder with no file within if I use -d, and by using -b it generates a markdown book with a chapter_1.md file with only one line "# Chapter 1"

**Comment by JoshBrudnak** [comment]
What command and OS are you using to run lojidoc? So far it has only been tested on Linux.

**Comment by jdemieville** [comment]
Ah, I see. I am using Windows OS so that could be the problem. I have been using the example usage from the readme with the appropriate path and repository URL lojidoc [Project_Path] -c [repository URL] -d docs/

**Comment by jdemieville** [comment]
It will flag if the directory is wrong with either incorrect file path or no java files found, depending on which is wrong. It is also making the markdown file or the doc folder (depending on which flag I've selected), so that seems to be working correctly. Could it be a consequence of my java program? The folder contains 4 java files, the main class and 3 other classes.

**Comment by JoshBrudnak** [comment]
No that shouldn't be the issue as long as your java syntax is correct. Is there any indication where in the application it is crashing (My guess would be during the file IO)? I would test it too but I don't have access to a windows computer right now :sob:

**Comment by jdemieville** [comment]
Here's the repository of the java program I was attempting to run: https://github.com/jdemieville/SherwoodVersusBinary

Maybe see if you can get it to work with this program on Linux. If not, it would be my program and I can test it with one of the java programs that you know it works with! 

I tried to run the rust backtrace it suggested, but it said the command was invalid. I was probably using it wrong!

**Comment by JoshBrudnak** [comment]
¯\\_(ツ)_/¯
![lojidoc](https://user-images.githubusercontent.com/9296527/46618393-5f50f080-caed-11e8-91d3-2459c194b406.png)

**Comment by jdemieville** [comment]
Ok, so it is either an operating system issue or I have done the commands incorrectly. When I get home, I will mimic the command that you used!

**Comment by jdemieville** [comment]
I ran the same commands an am getting the same error:

Generating documentation from SherwoodVersusBinary/src/

Keyword not supported: this
Keyword not supported: this
Keyword not supported: this
Keyword not supported: this
Keyword not supported: this
Keyword not supported: this
Keyword not supported: this
Keyword not supported: this
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 123, kind: Other, message: "The filename, directory name, or volume label syntax is incorrect." }', libcore\result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

Documentation finished. Generated 4 markdown files.

**Comment by jdemieville** [comment]
Sorry I can't get this example up for you! It's a cool concept!

**Comment by simbot912** [comment]
Picking this up — the 2018 attempt was blocked by a Windows path bug (`Os { code: 123 }` during file IO, which matches "only tested on Linux" per the maintainer), so I'll do the generation on Linux where it's known to work.

Concrete plan, keeping the PR small:
1. Build from source on Linux (`cargo build --release`).
2. Create a small sample Java project (3–4 classes with real javadoc comments — enough to show class/method docs, params, and returns without bloat) and commit it to `examples/sample-project/`.
3. Run `lojidoc examples/sample-project/ -d examples/generated-docs/` and commit the output.
4. Add a short "Example output" section to the README linking to `examples/generated-docs/`.

Two side notes from reading the thread:
- The repeated `Keyword not supported: this` warnings in @jdemieville's run look like a parser limitation worth a separate issue (it shouldn't silently skip `this` in javadoc-adjacent code).
- The Windows IO failure itself is probably a path-separator/drive-letter handling bug in the file walk — happy to file that too with a minimal repro if it still reproduces on current Windows.

I'll open the PR once the build passes; flagging here first in case you'd prefer the sample project to be an existing real repo instead of a small synthetic one.
